### PR TITLE
Allow Laravel 7

### PR DESCRIPTION
**Pull request by BertvanHoekelen**
Github action should succeed once https://github.com/christophrumpel/laravel-command-file-picker/pull/3 is merged.

- Changed github action matrix to match the one in [laravel-command-file-picker ](https://github.com/christophrumpel/laravel-command-file-picker/blob/master/.github/workflows/run-tests.yml) and added Laravel 7.
- Allow Laravel 6 and 7 in composer

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: run-tests
+
+on: [push]
```
Comment (christophrumpel) [inline .github/workflows/run-tests.yml]: Please also add the checks for pull_reuqest as it was before.

Comment (BertvanHoekelen) [reply .github/workflows/run-tests.yml]: Done!

**Comment by BertvanHoekelen** [comment]
@christophrumpel I think prefer-lowest doesn't really work. I think it tries to install `christophrumpel/laravel-command-file-picker` version 0.0.1 which doesn't have the `sortImports` method. Do you want me to remove it from the github action or do you have another idea?

**Comment by BertvanHoekelen** [comment]
I have changed it to use `prefer-stable` only, so that we can move forward. 😄We can always add `prefer-lowest` in a later PR or commit.

**Comment by christophrumpel** [comment]
Good catch. I was already wondering what's going on here. Thanks!

**Pull request merged**